src/plugins/CoreTests/src/BioStruct3DObjectTests.h

```diff
@@ -87,6 +87,17 @@ class GTest_BioStruct3DAtomResidueName : public XmlTest {
     int modelId;
 };
 
+class GTest_BioStruct3DMoleculeName : public XmlTest {
+    Q_OBJECT
+    SIMPLE_XML_TEST_BODY_WITH_FACTORY(GTest_BioStruct3DMoleculeName, "check-biostruct3d-molecule-name");
+
+    ReportResult report();
+
+    QString objContextName;
+    QString molName;
+    int chainInd;
```

Comment:
nit: use fully qualified names: 'Index', not 'Ind' when the length difference is not significant.